EpsilonDashboard/EpsilonDashboard.cpp

```diff
@@ -4,6 +4,8 @@
 #include "../PresenterLayer/PresenterContainer.h"
 #include "../ViewLayer/ViewContainer.h"
 
+#include <QCommandLineParser>
```

Comment:
Standard library includes go at the top, then third party libraries (this one), then internal files.